src/workers/personalizedDigestEmail.ts

```diff
@@ -213,6 +213,31 @@ const worker: Worker = {
       ...variationProps,
     };
 
+    const personalizedDigestWithLastSendDate = await con
+      .getRepository(UserPersonalizedDigest)
+      .findOne({
+        select: ['lastSendDate'],
+        where: {
+          userId: personalizedDigest.userId,
+        },
+      });
```

Comment:
This might not be enough because workers run in parallel and there could be a case where duplicate is sent so I think we will need some semafor mechanism to fully dedup.

Reply:
Hmm, probably very edge-casey, not sure if we should worry about that, but maybe Ido thinks differently about it.

Reply:
Achieving exactly once delivery is hard, this implementation already minimizes the chances for dups significantly. Let's leave it as it is now